store/wrapper/tmdb.go

```diff
@@ -0,0 +1,100 @@
+package wrapper
+
+import (
+	tdbm "github.com/cometbft/cometbft-db"
+	cdbm "github.com/cosmos/cosmos-db"
+)
+
+type DBWrapper struct {
+	db tdbm.DB
+}
+
+func NewCosmosDB(db tdbm.DB) cdbm.DB {
+	return &DBWrapper{db: db}
+}
+
+func (db *DBWrapper) Get(key []byte) ([]byte, error) {
+	return db.db.Get(key)
+}
+
+func (db *DBWrapper) Has(key []byte) (bool, error) {
+	return db.db.Has(key)
+}
+
+func (db *DBWrapper) Set(key []byte, value []byte) error {
+	return db.db.Set(key, value)
+}
+
+func (db *DBWrapper) SetSync(key []byte, value []byte) error {
+	return db.db.SetSync(key, value)
+}
+
+func (db *DBWrapper) Delete(key []byte) error {
+	return db.db.Delete(key)
+}
+
+func (db *DBWrapper) DeleteSync(key []byte) error {
+	return db.db.DeleteSync(key)
+}
+
+func (db *DBWrapper) Iterator(start, end []byte) (cdbm.Iterator, error) {
+	it, err := db.db.Iterator(start, end)
+	return it.(cdbm.Iterator), err
+}
+
+func (db *DBWrapper) ReverseIterator(start, end []byte) (cdbm.Iterator, error) {
+	it, err := db.db.ReverseIterator(start, end)
+	return it.(cdbm.Iterator), err
+}
+
+func (db *DBWrapper) NewBatch() cdbm.Batch {
+	return NewCosmosBatch(db.db.NewBatch())
+}
+
+func (db *DBWrapper) NewBatchWithSize(size int) cdbm.Batch {
+	return NewCosmosBatch(db.db.NewBatch())
+}
```

Comment:
This is the  db.Grow bug, we need to do batch with size

Reply:
Will wait a little longer to see if they do a release by end of day